conductor/submitter_nuke.py

```diff
@@ -223,6 +223,19 @@ def getOutputPath(self):
         output_path = file_utils.get_common_dirpath(write_paths)
         return output_path, write_paths
 
+    def checkSaveBeforeSubmission(self):
+        '''
+        Check if script has unsaved changes and prompt user if they'd like to
+        save it before continuing
+        '''
+        file_unsaved = nuke_utils.check_script_modified()
+        if file_unsaved:
+            title = "Unsaved Nuke Script Data"
+            message = "Save Nuke file script before submitting?"
```

Comment:
Can you change `Save Nuke file script before submitting?` to `Save Nuke script before submitting?`?